introduction.md

```diff
@@ -4,7 +4,7 @@
 
 ## **Large-scale automated source code refactoring**
 
-OpenRewrite enables large-scale distributed source code refactoring for framework migrations, vulnerability patches, and API migrations with an early focus on the Java language.
+OpenRewrite enables large-scale distributed source code refactoring for framework migrations, vulnerability patches, and API migrations with an early focus on the Java language ([see supported languages](/reference/supported-languages.md)).
```

Comment:
When I scan-read this sentence, even with the `see supported languages`, I might still walk away thinking there's no support for anything other than Java, which I don't think reflects our recent efforts. Can we update this to for instance say somehting along the lines of

> early focus on the Java language, and [continuously expanding language coverage](/reference/supported-languages.md)

Reply:
Yep will do - thanks!